### PR TITLE
Fix device grid bug - Change binding to use camelCase

### PR DESCRIPTION
**Pull request by handeadiguzel**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@handeadiguzel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: DeviceAdministration/Web/WebApiControllers/DeviceApiController.cs [hunk 1]
```diff
@@ -293,4 +293,12 @@ public DeviceApiController(IDeviceLogicND deviceLogic)
 #endif
 
     }
+
+    public class DataTablesResponseND
```
Comment (handeadiguzel) [inline DeviceAdministration/Web/WebApiControllers/DeviceApiController.cs]: fix this

File: DeviceAdministration/Web/Scripts/Views/Device/DeviceIndex.js [hunk 2]
```diff
@@ -370,6 +370,7 @@
 
         // create a success callback to track the selected row, and then call the DataTables callback
         var successCallback = function (json, a, b) {
+            console.log(arguments);
```
Comment (Inspiravetion) [inline DeviceAdministration/Web/Scripts/Views/Device/DeviceIndex.js]: remove please

**Comment by Inspiravetion** [comment]
lgtm after removing console log

File: DeviceAdministration/Web/Scripts/Views/Device/DeviceIndex.js (hunk 2 above)
Comment (handeadiguzel) [inline DeviceAdministration/Web/Scripts/Views/Device/DeviceIndex.js]: oops.. removed

**Pull request merged**